app/build.gradle

```diff
@@ -101,6 +101,9 @@ dependencies {
     implementation libraries.sheetMenu
 
     implementation libraries.aVLoadingIndicatorView
+    implementation libraries.stetho
+
+    implementation 'com.android.support:customtabs:27.0.2'
```

Comment:
since we're giving the customtabs version in dependencies.gradle, should this then be this?:
implementation libraries.customtabs